scripts/data.sql

```diff
@@ -1,7 +1,8 @@
 INSERT INTO jobs.users(upn, display_name) VALUES 
     ('s8182384', 'מיכאל הופמן'),
     ('s8258065', 'דניאל דימנשטיין'),
-    ('s8222765', 'אדיר סטיון');
+    ('s8222765', 'אדיר סטיון'),
+    ('SteveVaknin', 'סטיב וקנין');
```

Comment:
Your `upn` is not the expected one 🤔

Reply:
For our development in the open environment it is (the upn is like the id which we take from the github's authentication )